apps/web/src/hooks/use-auth.tsx

```
@@ -0,0 +1,63 @@
+'use client';
+
+import type { Session, User } from '@supabase/supabase-js';
+import { createContext, useContext, useEffect, useState } from 'react';
+import { createClient } from '@/lib/supabase/client';
+
+interface AuthContextType {
+  user: any | null;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)